src/components/app/data/utils.js

```diff
@@ -56,6 +56,33 @@ export function isSystemMaintenanceAlertOpen(config) {
   return true;
 }
 
+export const hasActiveLicenseOrLicenseRequest = ({ subscriptionPlan, subscriptionLicense, licenseRequests }) => (
```

Comment:
Do we wanna use these utility functions in the `useHasAvailableSubsidiesOrRequests` hook?

Reply:
+1, agreed there could likely be some consolidation here.